src/app/stack/recommender/recommender.component.ts

```diff
@@ -357,11 +361,20 @@ export class RecommenderComponent implements OnChanges {
         if (recommendations && length > 0) {
             for (let i: number = 0; i < length; ++ i) {
                 if (this.canCreateWorkItem(recommendations[i])) {
+                    console.log(recommendations[i]);
+                    let description: string = recommendations[i]['workItem']['message'];
+                    let codebase: any = recommendations[i]['workItem']['codebase'];
+                    description += '<br />';
```

Comment:
Yes. That's correct. But It is easily readable and updatable. If I need to just delete one line, this way it is easier.